ReactNative/ReactTwitter/android/app/src/main/java/com/reacttwitter/widgets/ReactButton.java

```diff
@@ -0,0 +1,50 @@
+package com.reacttwitter.widgets;
+
+import android.annotation.TargetApi;
+import android.content.Context;
+import android.graphics.drawable.Drawable;
+import android.os.Build;
+import android.support.v4.content.ContextCompat;
+import android.util.AttributeSet;
+import android.widget.Button;
+
+public class ReactButton extends Button {
+
+    public ReactButton(Context context) {
+        super(context);
+        init();
+    }
+
+    public ReactButton(Context context, AttributeSet attrs) {
+        super(context, attrs);
+        init();
+    }
+
+    public ReactButton(Context context, AttributeSet attrs, int defStyleAttr) {
+        super(context, attrs, defStyleAttr);
+        init();
+    }
+
+    @TargetApi(Build.VERSION_CODES.LOLLIPOP)
+    public ReactButton(Context context, AttributeSet attrs, int defStyleAttr, int defStyleRes) {
+        super(context, attrs, defStyleAttr, defStyleRes);
+        init();
+    }
+
+    private void init() {
+        setAllCaps(false);
+    }
+
+    public void setBackgroundImage(String backgroundImage) {
```

Comment:
The part about assets that have to be in the platform project is interesting. we should write about it in the doc